src/services/blocks/BlocksService.spec.ts

```diff
@@ -309,23 +281,43 @@ describe('BlocksService', () => {
 			// Reset LRU cache
 			cache.reset();
 
-			const historicApi = createApiWithAugmentations(
-				polkadotMetadataV29.toHex()
-			) as ApiDecoration<'promise'>;
+			/**
+			 * This is the mockApi adjusted to mock a runtime that uses
+			 * consts.system.blockWeights for its weight nomination.
+			 */
+			const mockHistoricApiAdjusted = {
```

Comment:
any reason for the term "Adjusted"?

Reply:
No particular reason, any suggestions are definitely welcome. I found the naming a little awkward so i stuck with adjusted